picard/ui/ui_options_metadata.py

```diff
@@ -2,16 +2,11 @@
 #
 # Created by: PyQt6 UI code generator 6.6.1
 #
-# Automatically generated - do not edit.
-# Use `python setup.py build_ui` to update it.
+# WARNING: Any manual changes made to this file will be lost when pyuic6 is
+# run again.  Do not edit this file unless you know what you are doing.
```

Comment:
This file has to be generated by running `python setup.py build_ui`

Reply:
```
# WARNING: Any manual changes made to this file will be lost when pyuic6 is
# run again.  Do not edit this file unless you know what you are doing.
```

is to be manually added, right ? running `python setup.py build_ui` doesn't seem to add that